### PR TITLE
Refurbish test suite

### PR DESCRIPTION
**Pull request by vext01**
This makes the Krun test suite work on newer versions of things found in Debian 10.

I don't think we've updated the deps since Debian 7 days!

I'll squash this all into one at the end.

### CODE REVIEW

File: krun/tests/test_linuxplatform.py [hunk 1]
```diff
@@ -439,6 +448,8 @@ def test_check_dmesg_filter0001(self, platform, mock_manifest):
             platform.get_dmesg_whitelist(), old_lines, new_lines, mock_manifest)
 
     def test_aslr0001(self, platform, caplog):
+        caplog.set_level(logging.DEBUG)
```
Comment (ltratt) [inline krun/tests/test_linuxplatform.py]: Is this intentional?

Comment (vext01) [reply krun/tests/test_linuxplatform.py]: Yeah. caplog used to capture all messages by default, but now it doesn't capture debug-level by default. Some tests check the debug level messages, so we have to turn them on.

Comment (ltratt) [reply krun/tests/test_linuxplatform.py]: OK, we should tweak the commit message a bit?

Comment (vext01) [reply krun/tests/test_linuxplatform.py]: Sure. I'll include it at squash time.

**Comment by ltratt** [comment]
Please squash / force push.

**Comment by vext01** [comment]
Give that a go.

**Comment by ltratt** [comment]
Works for me!

**Pull request merged**